doc/en/how-to/output.rst

```diff
@@ -36,6 +42,13 @@ option you make sure a trace is shown.
 Verbosity
 --------------------------------------------------
 
+Examples for modifying printing verbosity:
+
+.. code-block:: bash
+
+    pytest --quiet          # quiet - less verbose - mode
+    pytest -q               # quiet - less verbose - mode (shortcut)
+
 The ``-v`` flag controls the verbosity of pytest output in various aspects: test session progress, assertion
```

Comment:
It is a bit strange to list `-q` and `--quiet`, and right in the following paragraph discuss `-v`, which is not shown as example.

Please include `-v`, `-vv` and `-vvv` in the list above here as well.

Reply:
Will do. I'm sorry I just try to not add anything I wasn't told